__tests__/apps.test.js

```diff
@@ -195,3 +197,63 @@ describe("4. PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("5. GET /api/articles", () => {
+  test("status:200, returns all the articles if query is omitted", () => {
+    return request(app)
+      .get(`/api/articles`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeInstanceOf(Array);
+        expect(body.articles).toHaveLength(12);
+        let articles = body.articles;
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+  test("status:200, articles are sorted by date in descending order by default", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at")
```

Comment:
for this test we want to check that the articles are sorted by date when there is no ?sort_by added to the query, try removing this from the path and re-run the test to make sure it still passes

Reply:
the query should actually be "/api/articles?topic=mitch" or "/api/articles?topic=sport"